website/docs/packages/tutorial/js-module.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Lesson 2 - Create Reusable JS Modules
+hide_title: true
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Lesson 2 - Create Reusable JS Modules</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+A JavaScript module serves as a reusable code unit, encapsulating specific functionalities to promote an organized code structure. 
+
+This lesson will teach you how to create a JavaScript function that formats dates into a designated time zone. The function can be utilized across multiple applications, providing a consistent method for date formatting tailored to a particular time zone.
+
+
+By the end of this lesson, you will learn how to:
+
+* Create and configure the JS module 
+* Integrate and execute the module in your app
+
```

Comment:
Let's add a before you begin section - which tells them the basic understanding of how to create a package.